xbmc/video/VideoInfoTag.cpp

```diff
@@ -600,15 +642,42 @@ void CVideoInfoTag::ParseNative(const TiXmlElement* movie, bool prioritise)
   if (XMLUtils::GetString(movie, "sorttitle", value))
     SetSortTitle(value);
 
-  XMLUtils::GetFloat(movie, "rating", m_fRating);
+  const TiXmlElement* node = movie->FirstChildElement("ratings");
+  if (node)
+  {
+    for (const TiXmlElement* child = node->FirstChildElement("rating"); child != NULL; child = child->NextSiblingElement("rating"))
+    {
+      CRating r;
+      std::string name;
+      if (child->QueryStringAttribute("name", &name) != TIXML_SUCCESS)
+        name = "default";
+      XMLUtils::GetFloat(child, "value", r.rating);
+      XMLUtils::GetInt(child, "votes", r.votes);
+      int max_value = 10;
+      if ((child->QueryIntAttribute("max", &max_value) == TIXML_SUCCESS) && max_value >= 1)
+        r.rating = r.rating / max_value * 10; // Normalise the Movie Rating to between 1 and 10
+      AddRating(r, name);
+      bool isDefault = false;
+      if ((child->QueryBoolAttribute("default", &isDefault) == TIXML_SUCCESS) && isDefault)
+        m_strDefaultRating = name;
+    }
+  }
+  else
+  {
+    CRating r;
+    XMLUtils::GetFloat(movie, "rating", r.rating);
+	if (XMLUtils::GetString(movie, "votes", value))
+		r.votes = StringUtils::ReturnDigits(value);
+    int max_value = 10;
+    const TiXmlElement* rElement = movie->FirstChildElement("rating");
+    if (rElement && (rElement->QueryIntAttribute("max", &max_value) == TIXML_SUCCESS) && max_value >= 1)
+      r.rating = r.rating / max_value * 10; // Normalise the Movie Rating to between 1 and 10
+    AddRating(r, "default");
+    m_strDefaultRating = "default";
+  }
   XMLUtils::GetInt(movie, "userrating", m_iUserRating);
   XMLUtils::GetFloat(movie, "epbookmark", m_fEpBookmark);
   int max_value = 10;
-  const TiXmlElement* rElement = movie->FirstChildElement("rating");
```

Comment:
It already works like this. if i don't find ratings i search for rating and I set it as "default" rating